edge-hub/src/Microsoft.Azure.Devices.Routing.Core/RouteResult.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft. All rights reserved.

```

Comment:
Can you add unit tests for this class?

Reply:
I don't think such an UT would have anything meaningful. There's no real logic in this class, it's effectively a struct for all intents and purposes.  The IEquatable implementation was only added to facilitate updating the existing test collateral, and is not used anywhere else.

---
In reply to: [376164043](https://github.com/Azure/iotedge/pull/2469#discussion_r376164043) [](ancestors = 376164043)

Reply:
I think you need to implement unit test because:
1. even very simple logic can be incorrect; e.g. copy-and-paste and assign to wrong variable; string comparison whether is case sensitive or insensitive.
2. since code coverage is turned on, we should try to have code coverage for all code added/changed in order to increase our code coverage.